aiomcache/pool.py

```diff
@@ -16,9 +16,8 @@ def __init__(self, host, port, *, minsize, maxsize, loop=None):
         self._minsize = minsize
         self._maxsize = maxsize
         self._loop = loop
-        self._pool = asyncio.Queue(maxsize, loop=loop)
+        self._pool = asyncio.Queue(loop=loop)
```

Comment:
no need for the maxsize, control is done outside the data structure. Having a maxsize here is misleading and can also hide effects like creating more connections than needed (because they get queued or discarded)